### ISSUE TITLE
Swiss Chatter Boxes

### ISSUE THREAD

**Issue by barjacks** [jsoma/playfair-projects#154]
# Story:

Recently the right wing SVP were complaining about how French speakers spend too much time talking in parliament. 
# Data Source:

Developed a scraper to grab all the speeches of the members of the Swiss national council that are published after every parliamentary session three times a year and pull out the speakers name,their party, canton. I used the documents of the three sessions from 2016 from [here](https://www.parlament.ch/de/ratsbetrieb/amtliches-bulletin). Here's the scraper [code](https://github.com/barjacks/playfair-projects/blob/master/projects/barnaby_skinner/chatter_boxes/Scraping_PDF_Bulletin_2016.ipynb).
# Possible Headlines:
#### Right Wing SVP Don't Like Talking to the Media - Less so in Parliament
# Graphic:

![graphic_eng](https://cloud.githubusercontent.com/assets/1587242/17425243/1a332b06-5a99-11e6-9f0b-a288d01cdc45.png)
# Story issue checklist

PLEASE PASTE THIS AT THE END OF YOUR **STORY ISSUE POST**, AS THE FIRST COMMENT. I was wrong! You can delete this line after you paste it.

My pitch was (use the number): #76 
- [ ] My pitch has been approved (see `PITCHING.md`)
- [x] My story issue links to my pitch issue
- [x] I link to my finalized (ish) data source(s)
- [x] I've included a brief summary of my story
- [x] I've included some possible headlines or findings
- [x] I've included some links or images as inspiration (if you have any)
- [x] I have received two comments of peer feedback
- [x] I've included an update of my visualization/story in a comment
- [x] I have received two comments of peer feedback after posting an update
- [ ] I have received editorial feedback

**Comment by kromreig** [comment]
I would just like to say your regex is dope...

In the first bar graph on the top left, I think it might be clearer if you ordered the bars by descending/ascending talk time, though that might throw your colors.

It seems that purple/light purple are pretty hard to tell apart.

Also, my own implicit gender bias just can't seem to assign purple to male and blue to female! I know it's cliché, but in the interest of clarity, maybe reassess the colors? Also, can the abbreviations sit upright, like in the first bar? I think there's enough space.

Awesome awesome project.

**Comment by barjacks** [comment]
Thanks @kromreig I changed the bar graphs at the top, and you are right. Looks much better when they are sorted. And then I played around with the colors. On the top half I'm working with the color codes of the political parties. But I'm wondering, if I should give up on that - that way I'd be a bit more free picking colors. 

![graphic_eng](https://cloud.githubusercontent.com/assets/1587242/17462190/ce68c7cc-5c72-11e6-9aa6-11b91e5975d5.png)

**Comment by kromreig** [comment]
Love it! 👌

**Comment by palrogg** [comment]
Beautiful! I love the gender part!
Some thoughts:
* You write that female speak longer in general, but speak less than their male counterparts. Does it mean that the graphics on the left isn't balanced with the male/female ratio in each party? I think it would be good to explain wether it is or not
* If I remember well, you use an average of 6.5 seconds per word to calculate the speech durations. It seems important to me to specify that this isn't actual speech durations, but computed durations
* If the previous point is true, the calculation method may induct a bias, because german is an agglutinant language. Maybe the estimate should be different for each language
* Changing the 200, 400, … 1200 minutes ticks to hours would make it more readable

**Comment by barjacks** [comment]
Thank so much @palrogg.

Your thoughts and questions:
1. I've added more explaining text to address the fact that it's the SP and the BDP responsible for woman speeking longer than  men in average.
2. This too I have no explained in the graph. Does it  make sense?
3. According to this https://en.wikipedia.org/wiki/Words_per_minute in the majority of languages the reading speed is around 1000 characters per minute. I could time each politicians speeking time of course, with the videos that are online. Might be a little too much work though. But maybe for the top speakers it might still be worth looking into :-)
4. Good idea. I've changed the scale to hours. Looks much tidier.

![graphic_eng](https://cloud.githubusercontent.com/assets/1587242/17466469/c38c7b0c-5cde-11e6-8cc5-bea5f2546122.png)

**Comment by palrogg** [comment]
1 and 2: great, I think it's better to know how the calculation was made!
3 doesn't apply then, I didn't remember it was per character and not per word
4: perfect 👌

**Comment by mercybenzaquen** [comment]
This is looking awesome. I think the idea is great and you did a very good job breaking it down. My suggestions:
-Change fonts and maybe use one that you can use bold text and also plain text. Since you have a lot going on in your charts it could help to balance that out with a lighter font. Also it would help with the hierarchy.
-I know you chose the colors based on the parties' colors and I think that makes sense. I would maybe tone it down a bit. Maybe use transparency to make the colors more pale? Or look for their "pale" version lol. Right know you have a lot of bright colors in your first chart and then pale ones for the second one and it is a bit confusing.

Awesome job!!

**Comment by oargueso** [comment]
I would add the labels for the parties in the small gender pie charts. Also, the SVP -679 minutes is a little bit confusing, since it is below the graph it is supposed to be related too.

**Comment by jsoma** [comment]
Nice analysis!

**Colors:** I would ease back the colors a little bit, it's slightly aggressive. If you pick something from color brewer you might find some good combinations (or are these colors associated with the parties?). You also don't need to have each bar be a different color, since each is already a discrete bar. And... even if they are associated with the party, it's just soooo many colors that it gets slightly too busy for my tastes.

**Text:** Each time you have a title, I'd justify the subtitle right underneath it, and work on coming up with a consistent display of title + summary + chart. You're making good use of negative space but since it's so many graphics they end up being a little scattered layout-wise. "Sum of all the talks by political party" and the summary about how you calculated via division is probably important, but I would spend my text time telling a story and not explaining how the data was calculated.

**Time spent talking:** How did you find the "average time spent to read text" based on character length? I would probably put that in a note at the very bottom in a smaller font, citation-style.

**Font:** Is everything bold or is the font just a very heavy weight? I would try to ease them down a little bit so the graphics will pop a little more.

**Multiple images:** You could also split these into multiple images if you wanted! Give "multiple artboards" a google if you're interested.

**Comment by barjacks** [comment]
Thanks a lot for all your feedback and critiques. I have given up on using the colors of the parties. You are all rigth, just makes the charts to aggressive. And, heavy heartedly, I have changed my font. The font I had orginally chosen only came in bold. I've also rearranged the pie charts, and the subtitles of each graph. That also really makes things a lot tidier. 

![graphic_final](https://cloud.githubusercontent.com/assets/1587242/17493342/da956076-5d7d-11e6-8495-8c3bbc7faf4a.png)
